src/java/org/apache/cassandra/service/accord/AccordKeyspace.java

```diff
@@ -570,12 +553,12 @@ public static AccordCommand loadCommand(AccordCommandStore commandStore, TxnId t
         return command;
     }
 
-    private static <T> T deserializeWithVersionOr(UntypedResultSet.Row row, String dataColumn, String versionColumn, IVersionedSerializer<T> serializer, Supplier<T> defaultSupplier) throws IOException
+    private static <T> T deserializeWithVersionOr(UntypedResultSet.Row row, String dataColumn, LocalVersionedSerializer<T> serializer, Supplier<T> defaultSupplier) throws IOException
     {
-        if (!row.has(versionColumn))
+        if (!row.has(dataColumn))
```

Comment:
This isn't a regression as `null` writes to data column are no longer a null but `version:null`, so the data column will exist where it didn't before.